plugins/sampledata/blog/src/Extension/Blog.php

```diff
@@ -799,16 +799,17 @@ public function onAjaxSampledataApplyStep2()
         $menuTypes = [];
 
         for ($i = 0; $i <= 2; $i++) {
+
+            $title = $this->getApplication()->getLanguage()->_('PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MENUS_MENU_' . $i . '_TITLE');
```

Comment:
```suggestion
            $title = $this->getApplication()->getLanguage()->_('PLG_SAMPLEDATA_BLOG_SAMPLEDATA_MENUS_MENU_' . $i . '_TITLE');
```
Fix PHPCS empty line after for loop start